README.rst

```diff
@@ -96,7 +108,6 @@ you will need to select the *openwisp-monitoring* variant and *netjson-monitorin
     ./scripts/feeds update -a
     ./scripts/feeds install -a
     make menuconfig
-    # go to Base system, then select rpcd
```

Comment:
are you sure this is needed and are we sure this section of explainin how to select the packages with makemenu config is needed at all? The explanations above are enough in most cases, this seems redundant to me.

Reply:
This was not, that's why I removed it :sweat_smile:

Reply:
I added this section to make it consistent with `**openwisp-config**

Reply:
lol, I think it may be a good idea to remove it from there too then :grin: 

hey what do you think @nickberry17 @feckert, do you think this is useful at all?

Reply:
Would it be better to replace the section with a GIF?

![OpenWISP Package Compiling](https://user-images.githubusercontent.com/42766576/131209183-60b8308d-a283-43a7-bb3c-25fa3db9cfe5.gif)

Reply:
Yes, is always good to have something visual :+1: 
pelase upload the gif to a media branch (to keep the main branch lightweight).

Reply:
> Would it be better to replace the section with a GIF?
> 
> ![OpenWISP Package Compiling](https://user-images.githubusercontent.com/42766576/131209183-60b8308d-a283-43a7-bb3c-25fa3db9cfe5.gif)

I like the idea of a GIF, but it add a note saying it's not necessary to select all of those packages; only the one you need.